terraform/variables.tf

```diff
@@ -188,4 +188,9 @@ variable "generate_cost_report_lambda_zip" {
   type        = string
   description = "path to zip file containing lambda code to schedule cost report"
   default     = "../generate-cost-report-lambda/build/generate-cost-report-lambda.zip"
-}
\ No newline at end of file
+}
+
+variable "s3_backup_enabled" {
```

Comment:
We could do with a description here of what this means:

`Set to true if if the environment needs backing up e.g. in production. This will result in versioning and object locking being enabled for select buckets containing critical live data.`